addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/NumberedRenamableTabbedPane.java

```diff
@@ -56,6 +60,29 @@ public void stateChanged(ChangeEvent e) {
                         }
                     }
                 });
+
+        this.addMouseListener(
+                new java.awt.event.MouseAdapter() {
+                    @Override
+                    public void mouseClicked(java.awt.event.MouseEvent evt) {
```

Comment:
Do these need to be fully qualified or could they be imported?

Reply:
CNP code ;) Now fixed..